### DISCUSSION TITLE
HW 2 Part 4.3.b

### DISCUSSION THREAD

**Discussion by JimmySilentG** [cooperrc/computational-mechanics#92] [Q&A]
This is about the very last question of the HW. I have included a screenshot of what I propose to be the correct approach but the monte carlo method still isn't fully set in stone in my own understanding so maybe there is a better way of going about doing this.
In my code I am able to successfully calculate the mean and standard deviation of the buckling load but to find the length the beams should be so only a specified percentage reach a critical buckling load I am using a guess and check method along with properties associated with a normal distribution to basically say that if I place the 9,810 N load approximately 1.96 standard deviations above the mean this will satisfy the equation. Obviously because of rng the exactly value will change so this is what makes me suspect this is an acceptable method.
![image](https://user-images.githubusercontent.com/105613899/191317321-63292415-10f9-4010-899d-eb3925138fd4.png)

**Comment by cooperrc** [comment]
If you increase `N` in the function, the mean and std won't change as much. Monte Carlo methods converge with more random numbers generated.
